### PR TITLE
Update pathways.md

### PR DESCRIPTION
**Pull request by NomeQ**
Adds information from https://docs.google.com/document/d/14jVFJZUn1HGOOr5LA9XKh63d-XgbUWgKF-V0__Sbh9g/edit to examples/pathways. 

As discussed in a previous meeting, this may not be the final home for this content and it may be relocated as the organization of the site is considered.

### CODE REVIEW

File: docs/schedule/examples/pathways.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
 <hr>
 
+**Learn more: [Understanding Transit Accessibility](#understanding-transit-accessibility)**
```
Comment (isabelle-dr) [inline docs/schedule/examples/pathways.md]: I think the page is short enough that we don't need to reference different paragraphs. We can remove

File: docs/schedule/examples/pathways.md [hunk 2]
```diff
@@ -88,3 +90,67 @@ underground_walkway2,96,95,1,1
...
+
+[Examples: Text-to-Speech](https://gtfs.org/schedule/examples/text-to-speech/)
+
+While `tts_stop_name` is currently the only text-to-speech field officially adopted within the GTFS spec, other fields have been discussed and may be added. These include `tts_agency_name`, `tts_route_short_name`, `tts_route_long_name`, `tts_trip_headsign`, `tts_trip_short_name`, and `tts_stop_headsign`.
+
+Riders will need to use an app that supports text-to-speech functionality, in order to benefit from this information. Some apps, such as [NaviLensGo]((https://www.navilens.com/en/)), are designed specifically to assist riders with vision impairments to navigate stations and find the right vehicle. 
+
+### Adding Physical Accessibility Information about a Stop
```
Comment (isabelle-dr) [inline docs/schedule/examples/pathways.md]: I think here it's about stations, not stops

File: docs/schedule/examples/pathways.md [hunk 3]
```diff
@@ -88,3 +90,67 @@ underground_walkway2,96,95,1,1
...
+
+GTFS-Pathways is a component of GTFS that has the goal of providing more accurate information for riders about transit stations. GTFS-Pathways allows a rider to view a possible trip and understand whether they will be able to make a required transfer at a transit station. This information is currently considered recommended but not required within the GTFS spec, but this could change in the future.
+
+GTFS-Pathways adds the files `pathways.txt` and `levels.txt` as well as adding the `location_type` field in `stops.txt` to link together information described in pathways.txt 
+
+* [Reference: pathways.txt](../../reference/#pathwaystxt)
+* [Examples: GTFS-Pathways](#pathways)
```
Comment (isabelle-dr) [inline docs/schedule/examples/pathways.md]: This just links to this same page, I'd remove these three links.

**Comment by isabelle-dr** [comment]
@NomeQ thanks for this awesome contribution! 
To improve the flow on this page a bit, I'd recommend organizing it as follows:

**Title**: Accessibility and Pathways
**Sections**
## Why Display Accessibility Information?
## Accessibility Checklist
## Adding Wheelchair Accessibility in GTFS
## Adding Physical Accessibility Information about a Station
- the intro text you've drafted "GTFS-Pathways is a component of GTFS..."
### Describe the location of station entrances and exits
- pre-existing content
### Describe stairs and escalators (pre-exiting content)
- pre-existing content
### Describe elevators and pathways (pre-exiting content)
- pre-existing content

- add image at the end here -

**Comment by NomeQ** [comment]
Updated! Added a table of contents as well just so it was easy to skip directly to the pathways examples.

**Review by isabelle-dr — APPROVED** [review]
LGTM, thank you!

**Pull request merged**